purchase_order_type/README.rst

```diff
@@ -1,6 +1,8 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+===================
 Purchase Order Type
 ===================
 
```

Comment:
Include in the configuration section that you can predefine a purchase order type in the partner.

Include in the usage section that the type will be autofilled when selecting a partner that has a type set in its data.